geth/transactions/transactor_test.go

```diff
@@ -385,41 +386,72 @@ func (s *TxQueueTestSuite) TestLocalNonce() {
 	s.Equal(uint64(nonce)+1, resultNonce.(uint64))
 }
 
-func (s *TxQueueTestSuite) TestContractCreation() {
-	key, _ := crypto.GenerateKey()
-	testaddr := crypto.PubkeyToAddress(key.PublicKey)
-	genesis := core.GenesisAlloc{
-		testaddr: {Balance: big.NewInt(100000000000)},
-	}
-	backend := backends.NewSimulatedBackend(genesis)
-	selectedAccount := &account.SelectedExtKey{
-		Address:    testaddr,
-		AccountKey: &keystore.Key{PrivateKey: key},
-	}
-	s.manager.sender = backend
-	s.manager.gasCalculator = backend
-	s.manager.pendingNonceProvider = backend
-	tx := SendTxArgs{
-		From:  testaddr,
-		Input: hexutil.Bytes(gethcommon.FromHex(contract.ENSBin)),
+func (s *TxQueueTestSuite) TestTransactionsWithSimulatedBackend() {
+
+	testCases := []struct {
+		name             string
+		txInput          []byte
+		receiptValidator func(receipt *types.Receipt, testaddr common.Address)
+	}{
+		{
+			"EthTransfer",
+			hexutil.Bytes(gethcommon.FromHex("0.1")),
```

Comment:
i dont think that eth transfer has any input in tx

Reply:
My bad! Needed that in the .Value field. Pushing fix now